manager/csi/manager_test.go

```diff
@@ -4,11 +4,10 @@ import (
 	"context"
 	"net"
 
+	"github.com/docker/go-events"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	. "github.com/onsi/gomega/types"
-
-	"github.com/docker/go-events"
+	"github.com/onsi/gomega/types"
```

Comment:
Changed it from a `.` import to a regular input (it was only used in a single place here)